docker-machine-nfs.sh

```diff
@@ -135,7 +135,7 @@ setPropDefaults()
   prop_machine_name=
   prop_shared_folders=()
   prop_nfs_config="-alldirs -mapall="$(id -u):$(id -g)
-  prop_mount_options="noacl,async,nfsvers=3"
+  prop_mount_options="noacl,async,vers=3"
```

Comment:
why did you change this?

Reply:
Think you closed this by mistake?!

Reply:
It seems, that `nfsvers=` and `vers=` has the same [behavioural](http://manpages.ubuntu.com/manpages/bionic/man5/nfs.5.html). 

Will remove the change anyway,